stream/rhcos/rhcos.go

```diff
@@ -0,0 +1,16 @@
+package rhcos
+
+// Extensions is data specific to Red Hat Enterprise Linux CoreOS
+type Extensions struct {
+	AzureImage *AzureImage `json:"azure-image,omitempty"`
+}
+
+// AzureImage represents an Azure cloud image.
+type AzureImage struct {
+	// Release is the source release version
+	Release string `json:"release"`
+	// SourceManagedImage is the URL to an image already stored in Azure infrastructure
```

Comment:
Is this supposed to read

```suggestion
	// URL to an image already stored in Azure infrastructure
```

?

Reply:
Good catch @jlebon